### ISSUE TITLE
dropout in resnet.py

### ISSUE THREAD

**Issue by uniquezhengjie** [yangze0930/NTS-Net#14]
thinks for your great work, I found a err in resnet.py #line 148

` x = nn.Dropout(p=0.5)(x)`

set net.eval() doesn't work for it ,i think need to defind a self.drop with nn.Dropout(p=0.5) in __init__ function or change to F.dropout(x,p=0.5,training=self.training)

**Comment by paramansh** [comment]
I am facing the same issue. @uniquezhengjie does your solution work?

**Comment by DiKorsch** [comment]
This should be fixed since the model returns at test time different results! Took me an hour to figure it out. Should have checked the issues before :-/
